api/handlers/listhandler.py

```diff
@@ -607,19 +610,64 @@ def _check_ticket_for_batch(self, ticket):
     def put(self, *args, **kwargs):
         raise NotImplementedError("an analysis can't be modified")
 
+    def _default_analysis(self):
+        analysis_obj = {}
+        analysis_obj['_id'] = str(bson.objectid.ObjectId())
+        analysis_obj['created'] = datetime.datetime.utcnow()
+        analysis_obj['modified'] = datetime.datetime.utcnow()
+        analysis_obj['user'] = self.uid
+        return analysis_obj
+
     def post(self, cont_name, list_name, **kwargs):
         _id = kwargs.pop('cid')
         container, permchecker, storage, mongo_validator, _, keycheck = self._initialize_request(cont_name, list_name, _id)
-
         permchecker(noop)('POST', _id=_id)
+
+        if self.is_true('job'):
+            if cont_name == 'sessions':
```

Comment:
In the requirements when talking with @elisapoquette. I would say it's a temporary requirement that job-based analyses are only at the session level, but I defer to her for if this is a lasting requirement or something for the first pass.